includes/qliro-one-functions.php

```diff
@@ -70,6 +70,11 @@ function qliro_one_maybe_create_order() {
  * @return string|null
  */
 function qliro_wc_get_snippet() {
+
+	if ( ! qliro_is_enabled_with_demo_check() ) {
+		return;
```

Comment:
Issue: You should return null instead of void here. It probably is fine like this but its better to follow the expected return from the block comment.